lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,186 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"__main__.py", "qutebrowser"}}
```

Comment:
I am getting these:
```
class: qutebrowser
name: DuckDuckGo - qutebrowser
instance: qutebrowser
```

Is instance also not `qutebrowser` for you?

Matching `name` with `- qutebrowser` at the end might also work?!

btw: found this in the source / changelog
> - New `--qt-name` argument to qutebrowser which is passed to Qt to set `WM_CLASS`.